### ISSUE TITLE
Issues reported by ShapeChecker

### ISSUE THREAD

**Issue by ndjc** [oslc-op/oslc-specs#46]
The updated ShapeChecker reports quite a few errors on our vocabs and shapes. I have fixed the most obvious and simple ones, but some are larger-scale fixes. I have drafted some (but not all) of those in the branch 'larger_fixes_for_SC_errors'.

**Comment by jamsden** [comment]
Can you summarize the types of issues and are they in core and/or domains?

**Comment by ndjc** [comment]
I've already fixed the ones in core, cm, and config, since those were minor.  The new branch contains some possible larger changes in other domains.

I've attached a file with the complete ShapeChecker output on all domains, including core.
[errors.txt](https://github.com/oslc-op/oslc-specs/files/3277462/errors.txt)

**Comment by jamsden** [comment]
Ok. Some of those errors are on specifications that have not received any review and are low priority. We can address those specs based on demand/need. I'm not planning on doing a lot of work on specs no-one is using.

**Comment by berezovskyi** [comment]
Thanks a lot for your work, Nick! Especially for keeping the CLI args for the future use!

Jim, it's our choice which checks to require for the build to pass. I would suggest to start with failing the build in case of errors in the core vocab to begin with?

**Comment by jamsden** [comment]
We probably need some additional process here. Its fine for the build to fail if there are any errors in the vocab, we don't want to publish vocabs with errors to open-services.net. 

But we also probably don't want to publish any vocab or shapes files to open-services.net until we have at least an project specification revision through public review. 

Then we need to figure out what that publishing process is, and when it should be done.

**Comment by jamsden** [comment]
Let's discuss as we get closer to having QM and Core finished their public review, and we can vote to publish a project specification revision that is a candidate for OASIS standard.

**Comment by ndjc** [comment]
I agree with not yet working on the specs with no current effort - and that's why the new branch contains only some suggested changes for am, qm, and rm.
Core, cm, and conf have already had all real issues fixed - what's left are false positives or incorrect/out-of-date dcterms:source references that we still need to fix up.

**Comment by berezovskyi** [comment]
Merged the branch a while ago. Thanks for your work!
